.github/workflows/pep8-check.yml

```diff
@@ -0,0 +1,26 @@
+name: PEP8 Check
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  pep8_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: pip install autopep8
```

Comment:
Looking at https://github.com/DavidBergevoet/Setup-Reservation/blob/main/.github/workflows/python-app.yml, pip packages are being cached. It seems that `actions/setup-python` is not used here, so I'm not 100% sure if that is possible here. Could you check?

Reply:
I changed it so this will also use the setup-python, is this what you meant?